src/test/java/com/laserfiche/api/client/unit/AccessKeyTest.java

```diff
@@ -0,0 +1,53 @@
+package com.laserfiche.api.client.unit;
+
+import com.laserfiche.api.client.model.AccessKey;
+import com.nimbusds.jose.jwk.JWK;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.Objects;
+
+class AccessKeyTest {
+
+    @Test
+    void CreateFromBase64EncodedAccessKey_ValidBase64() throws Exception {
+        //Arrange
```

Comment:
While not strictly required, but would be nice if there a space between `//` and the text. See: https://google.github.io/styleguide/javaguide.html

Reply:
ok done along with fixing some code smells